sketch_oct9a.ino

```diff
@@ -201,7 +180,98 @@ void loop() {
   if (alt.getStatus() == Altimeter::ACTIVE) {
     last_alt = alt.getAltitude();
   }
+}
+
+void sendDataToRadio() {
+  Frame f = createFrame(
+    last_coords,
+    last_accel,
+    last_gyro,
+    last_alt
+#if CAPSULE == 2
+    ,
+    last_voc,
+    last_humid,
+    last_temp
+#endif
+  );
+
+  // In theory, this should send the data to the radio.
+  // In practice, this is completely untested. If nothing is connected to the UART pins, however,
+  // this doesn't do anything, so it's harmless to leave in for now.
+  Serial1.write(
+    (char*)&f,
+    FRAME_SIZE()
+  );
+}
+
+void setup() {
+  Serial1.begin(230400);
 
+#if CAPSULE == 2
+  // Pin A6: analog input from VOC sensor
+  pinMode(A6, PinMode::INPUT);
+#endif
+  // Pin D5: SD LEDs
+  pinMode(5, PinMode::OUTPUT);
+  // Pin D6: GPS LEDs
+  pinMode(6, PinMode::OUTPUT);
+  // Pin D7: sensor LEDs
+  pinMode(7, PinMode::OUTPUT);
+
+  pinPeripheral(0, PIO_SERCOM);
+  pinPeripheral(1, PIO_SERCOM);
+  pinPeripheral(8, PIO_SERCOM);
+  pinPeripheral(9, PIO_SERCOM);
+  pinPeripheral(A1, PIO_SERCOM);
+  pinPeripheral(A2, PIO_SERCOM);
+  pinPeripheral(A3, PIO_SERCOM);
+  pinPeripheral(A4, PIO_SERCOM);
+
+  // Turn on the error LEDs until initialization finishes
+  digitalWrite(5, LOW);
+  digitalWrite(6, LOW);
+  digitalWrite(7, LOW);
+
+  // Wait for a command from the radio. Sometimes the sensors need multiple tries, and calling
+  // initialize too many times does nothing, so put the initialize call in the loop.
+  do {
+    initializeAll();
+
+    if (Serial1.available()) {
+      //The string returned from readStringUntil does not include the terminator
+      String command = Serial1.readStringUntil('\n');
+
+      if (command == "wake") {
+        break;
+      } else if (command == "status") {
```

Comment:
the command name 'status' kinda makes me think of a sanity check, and not necessarily actually getting data. If you guys agree, do we want to change the name of the command to 'getInfo' or something more like that?

Reply:
These command names were Carson's idea. We should discuss this in the Discord.